src/mixins/itext_key_behavior.mixin.js

```diff
@@ -4,12 +4,17 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
    * Initializes hidden textarea (needed to bring up keyboard in iOS)
    */
   initHiddenTextarea: function() {
+    var text = this.canvas.getActiveObject();
```

Comment:
Why you have to cache this.canvas.getActiveObject(); when text at the end is 'this'?